docs/src/code-examples/sending-texts/go.mdx

```diff
@@ -0,0 +1,91 @@
+---
+lang: go
+---
+
+```go
+// resp, err := client.SendSMS(templateID, phoneNumber, ...options)
```

Comment:
Remove commented out code? Or, if you're trying to show use of options you could do this:

```go
options := []notify.WhateverTheOptionTypeIs{
  notify.Personalisation{
    {"name", userName},
  }
}

resp, err := client.SendSMS(templateID, phoneNumber, options...)
```

Untested code.